### PR TITLE
Add option for sequential test runs

### PR DESCRIPTION
**Pull request by MartynasStrazdas**

### CODE REVIEW

File: packages/components/package.json [hunk 1]
```diff
@@ -36,8 +36,10 @@
     "clean": "rimraf lib build",
     "cover": "nyc npm -s test",
     "lint": "eslint ./src/**/*.{ts,tsx}",
-    "test": "mocha --enable-source-maps --config ./.mocharc.json",
+    "test": "mocha --enable-source-maps --parallel --config ./.mocharc.json",
+    "test:dev": "mocha --enable-source-maps --config ./.mocharc.json",
     "test:watch": "npm -s test -- --reporter min --watch-extensions ts,tsx --watch",
+    "test:watch-dev": "npm run -s test:dev -- --reporter min --watch-extensions ts,tsx --watch",
```
[the remark this answers is not in the document]
Comment (grigasp) [reply packages/components/package.json:42]: same applies for `hierarchies-react/package.json` change

**Review by grigasp — COMMENTED** [review]
I think we need `test:dev` scripts for all other packages too, otherwise I'm going to forget which one to use when working on different packages at the same time :|

File: packages/components/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (MartynasStrazdas) [reply packages/components/package.json:42]: I don't think that is correct npm test should run our defined test script which would be the parallel one.
image

[the remark this answers is not in the document]
Comment (grigasp) [reply packages/components/package.json:42]: What I meant is that we don't need `watch` and `watch:dev`. Instead, we should keep only `watch`, which runs in non-parallel mode.

[the remark this answers is not in the document]
Comment (MartynasStrazdas) [reply packages/components/package.json]: changed

**Review by grigasp — APPROVED** [review]

**Pull request merged**